src/db/connection.js

```diff
@@ -0,0 +1,16 @@
+// Include Dotenv to load environments - will be used for db connection 
+require('dotenv').config()
```

Comment:
Raessa, please remove line 2 before merge. With the way that dotenv works, loading .env files from the top level, this is not needed.